akka-actor/src/main/scala/akka/dispatch/AbstractDispatcher.scala

```diff
@@ -23,7 +23,10 @@ import akka.event.EventStream
 import akka.event.Logging.{ Debug, Error, LogEventException }
 import akka.util.{ unused, Index, Unsafe }
 
-final case class Envelope private (message: Any, sender: ActorRef)
+final case class Envelope private (message: Any, sender: ActorRef) {
+  // TODO DOTTY -> bug? copy method becomes private with private constructor
+  def copy(message: Any, sender: ActorRef) = Envelope(message, sender)
```

Comment:
In Dotty the `copy` method of `case class`es have the same visibility as the constructor AFAIU.
Is it a deliberate change from Scala 2 to Scala 3 or is it accidental? @nicolasstucki @smarter

Reply:
see https://github.com/lampepfl/dotty/pull/5472

Reply:
Thanks @SethTisue ! Means we have to declare them explicitly.